tests/Feature/DisplaySingleJobOfferTest.php

```diff
@@ -38,6 +38,7 @@ public function test_display_a_single_job_offer(): void
             'minimumSalary' => 4000,
             'maximumSalary' => 4500,
             'status' => JobOffer::STATUS_APPROVED,
+            'hiring_type' => JobOffer::HIRING_TYPE_CLT,
```

Comment:
Why is this field snake_case? Is it passing on the tests?

Reply:
yes

PHPUnit 7.5.18 by Sebastian Bergmann and contributors.

Testing Project Test Suite
............................                                      28 / 28 (100%)

Time: 25.89 seconds, Memory: 62.50 MB

OK (28 tests, 42 assertions)


But I fixed